Remotion/Web/Development.WebTesting/WebDriver/Configuration/Chrome/ChromeUserDirectoryCleanUpStrategy.cs

```diff
@@ -0,0 +1,124 @@
+﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+// 
+// The re-motion Core Framework is free software; you can redistribute it 
+// and/or modify it under the terms of the GNU Lesser General Public License 
+// as published by the Free Software Foundation; either version 2.1 of the 
+// License, or (at your option) any later version.
+// 
+// re-motion is distributed in the hope that it will be useful, 
+// but WITHOUT ANY WARRANTY; without even the implied warranty of 
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the 
+// GNU Lesser General Public License for more details.
+// 
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+//
+using System;
+using System.IO;
+using System.Threading;
+using JetBrains.Annotations;
+using log4net;
+using Remotion.Utilities;
+
+namespace Remotion.Web.Development.WebTesting.WebDriver.Configuration.Chrome
+{
+  /// <summary>
+  /// Responsible for removing any files or directories added to the user directory during a test run.
+  /// </summary>
+  public class ChromeUserDirectoryCleanUpStrategy : IBrowserSessionCleanUpStrategy
```

Comment:
I don't know what was remarked in the last version of the review or changed since the last review.

Reply:
The `-Strategy` suffix has been added, the `Chromium-` prefix was changed to `Chrome-`.